### PR TITLE
Fix autofill for full name input row

### PR DESCRIPTION
**Pull request by neil-marcellini**
cc @luacmartins 
### Details

Fix autofill for first and last name on the profile page and the request a call modal. For some reason first and last name fill separately on the profile page and they fill simultaneously on the request a call modal. I'm not sure why that is, but I think any autofill is better than none.
### Fixed Issues

$ https://github.com/Expensify/App/issues/7529

### Tests

First set up autofill:
 - For Chrome go to chrome://settings/addresses
 - For Mobile Web Safari go to Settings > Safari > Autofill and turn on 'Use Contact Info'

1. Add an address and enter a first and last name in the name field. Add a phone number including a country code (e.g, +1-945-881-9540).

Then test autofill on the profile page and the request a call modal.

1. Create a new account on new expensify.
2. Go to Settings > Profile.
3. Click into the first name field.
4. Make sure you see an autofill prompt like so:
image

5. Click the autofill prompt and verify that the first name is entered.
6. Click into the last name field
7. Click the autofill prompt and verify that the last name is entered.
8. Click save and make sure it is successful.
9. Create a chat with concierge.
10. Click the call button in the upper right.
11. Click into the first name field and make sure you see an autofill prompt.
12. Click the autofill prompt and verify that the first name, last name, and phone number are entered.
13. Click 'Request a Call' and make sure that it is successful.

- [ ] Verify that no errors appear in the JS console

### QA Steps

Same as the test steps above.
- [ ] Verify that no errors appear in the JS console

### Tested On

- [x] Web
- [x] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

#### Web

https://user-images.githubusercontent.com/26260477/155400538-6b9404c1-4e97-4f57-8b2e-23beacb0c1d2.mov

#### Mobile Web

https://user-images.githubusercontent.com/26260477/155401074-ec51fa75-aca3-4778-b15a-cb6f0ee73107.mp4

#### Desktop

image

#### iOS

image

#### Android

image

### CODE REVIEW

File: src/components/FullNameInputRow.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ const FullNameInputRow = (props) => {
             <View style={styles.flex1}>
                 <TextInput
                     label={props.translate('common.firstName')}
+                    name="fname"
```
Comment (nickmurray47) [inline src/components/FullNameInputRow.js:51]: do we use this `name` prop at all? I don't see it come up in the `BaseTextInput` component

Comment (luacmartins) [reply src/components/FullNameInputRow.js:51]: I think this is what the browser uses to autofill the input.

**Review by luacmartins — APPROVED** [review]

File: src/components/FullNameInputRow.js (hunk 1 above)
Comment (nickmurray47) [reply src/components/FullNameInputRow.js:51]: Ah, found where it's being used but still don't totally understand what it's for https://github.com/Expensify/App/blob/1ab42aba7527e201fda9784b791d371fdc479e98/src/components/TextInput/index.js#L13-L16

Maybe some follow-up documentation would be helpful @neil-marcellini

**Review by nickmurray47 — APPROVED** [review]

File: src/components/FullNameInputRow.js (hunk 1 above)
Comment (neil-marcellini) [reply src/components/FullNameInputRow.js:51]: Yeah as Carlos said it's what the browser uses to autofill the input. Here is the reference I followed https://developers.google.com/web/fundamentals/design-and-ux/input/forms#recommended_input_name_and_autocomplete_attribute_values
Sorry that I didn't see this yesterday.

**Comment by mountiny** [comment]
Seems like Applause could not verify this worked as expected: https://github.com/Expensify/App/issues/7982

**Comment by SumitDiyora** [comment]
**Issue 1 -**

**Title** - [Medium] Chrome: Color Contrast: Insufficient color contrast for text.
**Actual** - Background/foreground color combinations with an insufficient contrast ratio was found on text. For instance, the #7D8B8F "The current wait time is 0 minutes.  Please note that our Guides are typically available from Sunday at 5pm CT to Friday at 5pm CT.Call me" text has an insufficient color contrast ratio of 3.5.1 with a #FFFFFF background.
**Expected** - The "The current wait time is 0 minutes.  Please note that our Guides are typically available from Sunday at 5pm CT to Friday at 5pm CT.Call me" text should have a contrast ratio against the page background of 4.5:1 or greater.
**WCAG failure** - 1.4.3
**Reproducible in staging?:** - Yes
**Version Number:** - v1.1.40-2
**Platforms** - Web, Desktop, Mobile-web, iOS
**Suggested Fix** - Ensure that each text/background color combination presents a contrast ratio of at least 4.5:1.

**Note** - Due to the installation of the build not able to check on the Android.

#7826_Color contrast for text

**Issue 2 -**

**Title** - [Medium]: Chrome+Jaws : Insufficient color contrast for form controls.
**Actual** - Background/foreground color combinations with an insufficient contrast ratio was found on the screen. For instance, the #ECECEC border of "First name", "Last name", "Phone number" and "Extension (Optional)" form fields has a contrast ratio of 1.2:1 with the #FFFFFF background.
**Expected** - The border of the input field should have a contrast ratio against the page background of 3:1 or greater.
**WCAG failure** - 1.4.11
**Reproducible in staging?** - Yes
**Version Number:** - v1.1.40-2
**Platforms** - Web, Desktop, Mobile-web, iOS
**Suggested Fix** - Ensure that the color contrast for all user interface controls in their default as well as active, hover, and focus states is 3:1 with the adjacent content.

**Note** - Due to installation of build not able to check on 'Android'.

#7826_Color contrast for form field


**Issue 3 -**

**Title** - [High]: Chrome+Jaws : Screen reader: Role and label are not defined for "back" control.
**Actual** - Role is not defined for the "back (<)". interactive element. This made it difficult for screen reader users to access the functionality associated with it. Additionally, the label is not defined for the mentioned interactive element in the source code of the page.
**Expected** - When a role is defined appropriately for the interactive element, the screen reader user will interact with the element effectively.
**WCAG failure** - 4.1.2, 2.5.3
**Reproducible in staging?** - Yes
**Version Number:** - v1.1.40-2
**Platforms** - Web, Desktop, Mobile-web, iOS
**Suggested Fix** - Ensure that all the page functionality is available for all users irrespective of the device being used.
Apply the following changes:

- Specify role="button" for the 'div' element containing the "back (<)" control.
- Provide aria-label value such as "back" for the 'div' element containing the "back (<)" control.

**Note** - Due to installation of build not able to check on 'Android'.

#7826_Role and label not defined for back control

**Issue 4 -**

**Title** - [High]: Chrome+Jaws: Screen reader: Role is not defined for 'Call me' control.
**Actual** - Role is not defined for the "Call me" control. This made it difficult for screen reader users to access the functionality associated with it.
**Expected** - Role should be defined as a 'button' for the "Call me" control.
**WCAG failure** - 4.1.2
**Reproducible in staging?:** - Yes
**Version Number:** - v1.1.40-2
**Platforms** - Web, Desktop, Mobile-web, iOS
**Suggested Fix** - Ensure that all the page functionality is available for all users irrespective of the device being used.
Apply the following changes:

- Specify role="button" for the 'div' element containing the "Call me" control.

**Note** - Due to installation of build not able to check on 'Android'.

#7826_Role not defined for call me control

**Issue 5 -**

**Title** - [Medium] Chrome: Color Contrast: Insufficient color contrast for 'Call me' control.
**Actual** - Background/foreground color combinations with an insufficient contrast ratio was found on control. For instance, the #03D47C "Call me" control text has an insufficient color contrast ratio of 2.0.1 with a #FFFFFF background.
**Expected** - The "Call me" text control should have a contrast ratio against the page background of 4.5:1 or greater..
**WCAG failure** - 1.4.3
**Reproducible in staging?:** - Yes
**Version Number:** - v1.1.40-2
**Platforms** - Web, Desktop, Mobile-web, iOS
**Suggested Fix** - Ensure that each text/background color combination presents a contrast ratio of at least 4.5:1.

**Note** - Due to the installation of the build not able to check on the Android.

#7826_Color contrast for call me control

**Issue 6 -**

**Title** - [Medium] Chrome: Color Contrast: Insufficient color contrast for error message text.
**Actual** - Background/foreground color combinations with an insufficient contrast ratio was found on error message text (which becomes available when "Phone number field is empty"). For instance, the #FC3826 "Please enter a valid phone number without brackets or dashes. If you're outside the US please include your country code, eg. +447782339811" text has an insufficient color contrast ratio of 3.7.1 with a #FFFFFF background.
**Expected** - The "Please enter a valid phone number without brackets or dashes. If you're outside the US please include your country code, eg. +447782339811" error message text should have a contrast ratio against the page background of 4.5:1 or greater.
**WCAG failure** - 1.4.3
**Reproducible in staging?:** - Yes
**Version Number:** - v1.1.40-2
**Platforms** - Web, Desktop, Mobile-web, iOS
**Suggested Fix** - Ensure that each text/background color combination presents a contrast ratio of at least 4.5:1.

**Note** - Due to the installation of the build not able to check on the Android.

#7826_Color contrast for error message text

**Pull request merged**